bokehjs/test/codebase/size.ts

```diff
@@ -8,17 +8,15 @@ const build_dir = path.normalize(`${__dirname}/../..`) // build/test/codebase ->
 
 const LIMITS = new Map([
   // es2017
-  ["js/bokeh.min.js",                700],
-  ["js/bokeh-widgets.min.js",        300],
-  ["js/bokeh-tables.min.js",         270],
-  ["js/bokeh-api.min.js",             90],
-  ["js/bokeh-gl.min.js",              70],
+  ["js/bokeh.min.js",               750],
+  ["js/bokeh-widgets.min.js",       300],
+  ["js/bokeh-tables.min.js",        270],
+  ["js/bokeh-api.min.js",            90],
```

Comment:
Alignment is off.

Reply:
If there is going to be alignment at all, it should encompass both the "top" and "bottom" halves.